### PR TITLE
Draft Proposal: Selection Process for PyCon APAC 2024 Host

### PR DESCRIPTION
**Pull request by darjeeling**
This pull request introduces a draft document that delineates the procedure for selecting the host of PyCon APAC 2024. Your reviews and comments are greatly appreciated to help improve and finalize this document.

### CODE REVIEW

File: 2024/cfp-main-2024.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+  - [What is PyCon APAC](#what-is-pycon-apac)
+- [On-Site Team Requirements](#on-site-team-requirements)
+- [Proposal Structure](#proposal-structure)
+  - [APAC Organizers Progress Meetings](#apac-organizers-progress-meetings)
+- [Proposal Format](#proposal-format)
+  - [Samples of Proposals](#samples-of-proposals)
+  - [PyCon APAC - On the Longer Term](#pycon-apac---on-the-longer-term)
+  - [Additional considerations for PyCon APAC 2024](#additional-considerations-for-pycon-apac-2024)
+
+This is the announcement for the Call for Participation (CfP) for PyCon Asia Pacific
```
Comment (uranusjr) [inline 2024/cfp-main-2024.md]:
```suggestion
This is the announcement for the Call for Participation (CfP) of PyCon Asia Pacific
```

File: 2024/cfp-main-2024.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+PyCon APAC has gradually increased, leading to this Call for Participation. It is our
+hope that different Asia Pacific countries will take turns to host as the momentum of
+Python develops in this region.
+
+This Call for Participation is meant to collect proposals from teams wishing to help run
+PyCon Asia Pacific in 2024 in a location they are based in. PyCon APAC is
+intended to serve as a platform for Python users throughout Asia-Pacific countries to
+come together to meet with fellow Python users. This is also true for anyone who is exploring Python.
+
+If you have not done so, pease also follow [PyCon APAC Twitter](https://twitter.com/pyconapac) for more
```
Comment (uranusjr) [inline 2024/cfp-main-2024.md]:
```suggestion
If you have not done so, please also follow [PyCon APAC Twitter](https://twitter.com/pyconapac) for more
```

File: 2024/cfp-main-2024.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+- More than 2 parallel tracks, with at least 2 tracks in English
+
+Given the nature of PyCon APAC, we are open to accepting proposals which can only
+host fewer attendees, with a target of 100 attendees being the minimum.
+
+# On-Site Team Requirements<a name="onsite"></a>
+
+These are the requirements that an on-site team will sign-up to when submitting a
+proposal. Changes to these requirements are possible, but they must be approved
+ by the APAC before implementation.
```
Comment (uranusjr) [inline 2024/cfp-main-2024.md]:
```suggestion
by the APAC before implementation.
```

File: 2024/cfp-main-2024.md [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+
+1. The conference will be financially and legally run by an entity registered at the
+   hosting country and for which the on-site team is part of.
+2. The on-site team must be geographically located in a country in the Asia Pacific
+   and Australasia region.
```
Comment (uranusjr) [inline 2024/cfp-main-2024.md]:
```suggestion
   or Australasia region.
```

File: 2024/cfp-main-2024.md [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+3. The on-site team must be willing to actively coordinate with the PAC board and
+   the other regional Python groups or Python workgroups where applicable, so
```
Comment (uranusjr) [inline 2024/cfp-main-2024.md]:
```suggestion
   other regional Python groups or Python workgroups where applicable, so
```

File: 2024/cfp-main-2024.md [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+   3. Venue manager: handling all contacts with the venue and caterer
+   4. Logistics manager: handling all aspects of local logistics and customs
+   5. Conference desk manager: responsible for the on-site help desk, on-site registrations, tickets, cash refunds,
+      attendee support, etc.
+6. List of all team members that will be present during the conference itself as
+   volunteers (but not necessarily help before the conference begins), and/or sources from which people can be or
+   will be acquired if needed (e.g., universities, other local associations, etc.). This list is indicative,
+   as most of the volunteers helping during the conference are usually enrolled during the last months/weeks before
+   the conference.
+7. List of potential international and local sponsors. Note : Local sponsors help a
```
Comment (uranusjr) [inline 2024/cfp-main-2024.md]:
```suggestion
7. List of potential international and local sponsors. Note: Local sponsors help a
```

**Review by uranusjr — COMMENTED** [review]
Some typographic suggestions. I’ll add suggestions for the content in a later review.

File: 2024/cfp-main-2024.md [hunk 7]
```diff
@@ -0,0 +1,293 @@
...
+number of participants at PyCon APAC varies depending on the hosting country. The
+following is an estimate of past PyCon APAC's:
+
+- 350+ participants in PyCon APAC (Singapore)
+- 500+ participants in PyCon APAC (Tokyo, Japan)
+- 700+ participants in PyCon APAC (Taipei, Taiwan)
+- 1000+ participants in PyCon APAC (Seoul, South Korea)
+- 200+ participants in PyCon APAC (Kuala Lumpur, Malaysia)
+- 500+ participants in PyCon APAC (Manila, Philippines)
+- 100+ participants in PyCon APAC online (Bangkok, Thailand)
```
Comment (uranusjr) [inline 2024/cfp-main-2024.md]: Should we add the year to each item?

File: 2024/cfp-main-2024.md [hunk 8]
```diff
@@ -0,0 +1,293 @@
...
+   1. The venue must provide breakout rooms for the number of expected
+   attendees and the number of parallel tracks, with 2 English-based tracks
+   being the minimum.
+   2. Note: In past PyCon APAC's, on-site teams have sought and
+   collaborated with institutions of higher learning (IHL's) as they can
+   provide the venues required at no or low cost.
+7. The conference must provide the following services to all attendees. Proposals
+   will have to provide details about how these can be implemented at the
+   proposed venues:
+   1. Food and drinks for lunches and breaks on all conference days (and tutorials and sprints where applicable)
```
Comment (uranusjr) [inline 2024/cfp-main-2024.md]: I feel we should also ask the proposal to include dietary options to accomodate possible preferences. Making a list of options a hard requirement is likely too unreasonable, but we should encourage the organizers to investigate the possibilities.

File: 2024/cfp-main-2024.md [hunk 9]
```diff
@@ -0,0 +1,293 @@
...
+    1. "We can get additional support from the local government / city council"
+    2. "We need to provide tickets to city locals at reduced prices"
+    3. "We intend to serve meals in a different venue"
+    4. "We have some extra space available for posters / sponsors"
+    5. "We want to have our local Python conference run as satellite event" etc.
+12. Venue proposal. Describe the following sub-items:
+    1. Location, reachability by car, train, plane
+    2. Venue floor plans, catering floor plans (if different from venue)
+    3. Total capacity for talk rooms, training rooms, plenary sessions, booth
+       space and catering/restaurants; if possible, including room plans
```
Comment (uranusjr) [inline 2024/cfp-main-2024.md:214]: We should also ask for individual capacities of broken down rooms (or maybe this is the intended meaning?)

File: 2024/cfp-main-2024.md [hunk 10]
```diff
@@ -0,0 +1,293 @@
...
+    4. A/V equipment
+    5. Video recording, availability of on-site support for A/V recordings,
+       possibly even including live- or post-editing
+    6. Lightning equipment, availability of on-site support for lighting
```
Comment (uranusjr) [inline 2024/cfp-main-2024.md]: Is this supposed to be _Lighting_ equipments instead?

File: 2024/cfp-main-2024.md (hunk 9 above)
Comment (cr8ivecodesmith) [reply 2024/cfp-main-2024.md:214]: I feel like this means "as much detail as possible". We could rephrase this like so:

Provide detailed information on the number of people you can accommodate in lecture rooms, training spaces, main session halls, exhibition areas, and dining spots. If you can, include diagrams or layouts of these spaces.

File: 2024/cfp-main-2024.md (hunk 7 above)
Comment (iqbalabd) [reply 2024/cfp-main-2024.md]: Yes @uranusjr . Can I ask you to add them on each line?

File: 2024/cfp-main-2024.md (hunk 8 above)
Comment (iqbalabd) [reply 2024/cfp-main-2024.md]: Good idea. Can I ask you to add that too @uranusjr ?

File: 2024/cfp-main-2024.md (hunk 10 above)
Comment (iqbalabd) [reply 2024/cfp-main-2024.md]: Yes, seems like it. We've never had any outdoor conferences that require equipment for lightning. Please fix if you can.

Comment (uranusjr) [reply 2024/cfp-main-2024.md]:
```suggestion
    6. Lighting equipment, availability of on-site support for lighting
```

**Comment by uranusjr** [comment]
I put the addition of yearly participant and dietary information in a downstream pull request: https://github.com/darjeeling/pyconapac-proposals/pull/2

**Comment by Cheukting** [comment]
Hey thank you @darjeeling for making this PR, I have no extra comments 👍🏼

**Comment by dmaharika** [comment]
All is good, I think I have no extra comments 👍

**Comment by iqbalabd** [comment]
Looks good. Merging this.

**Pull request merged**